Formula/rtptools.rb

```diff
@@ -0,0 +1,31 @@
+class Rtptools < Formula
+  desc "Set of tools for processing RTP data"
+  homepage "https://www.cs.columbia.edu/irt/software/rtptools/"
+  url "https://www.cs.columbia.edu/irt/software/rtptools/download/rtptools-1.21.tar.gz"
+  sha256 "2ddbf6d3a4dbdef26ca85e860f6795264a359d30e24fd63f14a8c4f7ca331830"
+
+  def install
+    system "./configure", "--disable-debug",
```

Comment:
```
configure: WARNING: unrecognized options: --disable-debug, --disable-silent-rules
```

Reply:
I will update to rtptools 1.22 which should fix this